src/cli/README_DATASET.md

```diff
@@ -695,3 +696,74 @@ Done
 0e080000000000010000000300001635060004001fffe00208d196fa2040e973b60708fdbbc310c48f3a3905109929154dbc363218bcd22f907caf5c15030f4f70656e5468726561642d646532620102de2b041015b2c16f7ba92ed4bc7b1ee054f1553f0c0402a0f7f8
 Done
 ```
+
+### updater
```

Comment:
Should we also add an example to the Quick Start section? Maybe just after https://github.com/openthread/openthread/blob/main/src/cli/README_DATASET.md#attach-to-existing-network

Reply:
Good idea. Added a section with an example.